fixtures/test_rustscan_scripts.toml

```diff
@@ -0,0 +1,14 @@
+# Test/Example ScriptConfig file
+
+# Tags to filter on scripts. Only scripts containing all these tags will run.
+tags = ["core_approved", "example"]
```

Comment:
Are we over-complicating this tags system? Maybe we can ignore `tags` and `developer` completely until we actually allow people to contribute scripts. This will keep our code simpler and allow us to evolve it when we have the need to.